lib/data/constants/PETAL_CONSTANTS.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:meta/meta.dart';
+import 'dart:math';
+import 'package:flutter/material.dart';
+
+import 'package:learning_compass_exp/data/models/petal.dart';
+import 'package:learning_compass_exp/data/models/petal_names.dart';
+import 'package:learning_compass_exp/common/widgets/custom_icons_icons.dart';
+
+/// This file exports an instance of the below private class [_PetalConstants].
+/// Contains all implementation constants needed throughout the app.
+/// Might be split into multiple files later if needed.
+@immutable
+class _PetalConstants {
+  final Petal workLifeBalance;
+  final Petal safety;
+  final Petal lifeSatisfaction;
+  final Petal health;
+  final Petal civicEngagement;
+  final Petal environment;
+  final Petal education;
+  final Petal community;
+  final Petal job;
+  final Petal income;
+  final Petal housing;
+
+  _PetalConstants({
+    this.workLifeBalance,
+    this.safety,
+    this.lifeSatisfaction,
+    this.health,
+    this.civicEngagement,
+    this.environment,
+    this.education,
+    this.community,
+    this.job,
+    this.income,
+    this.housing,
+  });
+
+  Map<PetalName, Petal> toMap() {
+    return {
+      PetalName.workLifeBalance: workLifeBalance,
+      PetalName.safety: safety,
+      PetalName.lifeSatisfaction: lifeSatisfaction,
+      PetalName.health: health,
+      PetalName.civicEngagement: civicEngagement,
+      PetalName.environment: environment,
+      PetalName.education: education,
+      PetalName.community: community,
+      PetalName.job: job,
+      PetalName.income: income,
+      PetalName.housing: housing,
+    };
+  }
+
+  List<Petal> toList() {
+    return [
+      workLifeBalance,
+      safety,
+      lifeSatisfaction,
+      health,
+      civicEngagement,
+      environment,
+      education,
+      community,
+      job,
+      income,
+      housing
+    ];
+  }
+}
+
+
+
+
+@literal
+final _PetalConstants PETAL_CONSTANTS = _PetalConstants(
+  workLifeBalance: Petal(
+      name: PetalName.workLifeBalance,
+      stringName: 'Work-Life Balance',
```

Comment:
Would it be better if "stringName" would be a "title" instead? After all that's kind of what they will be used for.

Reply:
No. I think variables should be named descriptively, not prescriptively. Meaning the name of the variable should describe what it is, not what it's used for.

Reply:
Yes, I see what you mean. But we already have a variable "name" for a PetalName, so that obviously can not be used. Perhaps "title" is not the best alternative. Either way, I'm not sure it's good practice to prefix the name with the type of the variable.